misk-hibernate/src/main/kotlin/misk/hibernate/BulkShardMigrator.kt

```diff
@@ -0,0 +1,435 @@
+package misk.hibernate
```

Comment:
I'm not sure if this should be in the hibernate package. It's a specific workflow for applications to migrate data. Maybe misk.hibernate.migrate

Reply:
moved to `misk.hibernate.migrate`